### PR TITLE
adds sulrich crkbd (corne) personal keymap

### PR DESCRIPTION
**Pull request by sulrich**
this adds my personal keymap for the corne keyboard.  it is a reasonably simple mac os oriented keymap which should be reasonably intuitive for mac and vim users to be able to use.  

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/sulrich/config.h [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+#define USE_SERIAL_PD2
```
Comment (drashna) [inline keyboards/crkbd/keymaps/sulrich/config.h:31]: This is no longer used. 
```suggestion
```

File: keyboards/crkbd/keymaps/sulrich/keymap.c [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      KC_TRNS, KC_MPRV, KC_MPLY, KC_MNXT, KC_VOLD, KC_VOLU,                      KC_TRNS, KC_MUTE, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                          KC_TRNS, KC_TRNS,  KC_ENT,    KC_TRNS, KC_TRNS, KC_TRNS
+                                      //`--------------------------'  `--------------------------'
+  )
+};
+
+#ifdef OLED_DRIVER_ENABLE
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+  if (!is_master) {
```
Comment (drashna) [inline keyboards/crkbd/keymaps/sulrich/keymap.c:63]:
```suggestion
  if (!is_keyboard_master()) {
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/crkbd/keymaps/sulrich/config.h (hunk 1 above)
Comment (sulrich) [reply keyboards/crkbd/keymaps/sulrich/config.h]: thanks! resolved in the latest push.

File: keyboards/crkbd/keymaps/sulrich/keymap.c (hunk 2 above)
Comment (sulrich) [reply keyboards/crkbd/keymaps/sulrich/keymap.c]: thanks!  also updated the `is_keyboard_master()` later in the keymap.

**Comment by sulrich** [comment]
thanks for the review and the suggestions.  these have been integrated as well as a little more cleanup.

**Review by drashna — APPROVED** [review]

**Comment by sulrich** [comment]
hello - 

just bumping and noting that all requested changes have been made and approved.  please let me know if there’s anything else.

thanks!

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**